google-cloud-clients/google-cloud-bigquerystorage/pom.xml

```diff
@@ -41,6 +41,10 @@
             <artifactId>gax-grpc</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.cloud</groupId>
+            <artifactId>google-cloud-bigquery</artifactId>
+        </dependency>
```

Comment:
Is this dependency only added for testing?  If so, can you please add `<scope>test</scope>`?

Reply:
It is only needed for test. Forgot to adjust scope when I added it.

Done with commit: https://github.com/googleapis/google-cloud-java/pull/5242/commits/8c728a6e16a249baef728f80c384cdd5df97d9a5